style.css

```diff
@@ -0,0 +1,112 @@
+@import url("https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto&display=swap");
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: #170606;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+header {
+  width: 375px;
```

Comment:
[Optional - but highly recommended]

This fixed width is not needed. Actually, we should remove it. 
Think about it: there are different mobile screen sizes. We want our app to be responsive regardless of what type of mobile we use.

- [ ] If I would be you, this fixed width would be removed and I would take full advantage of Flexbox, making my app fully responsive regardless of the mobile type I would use. You can do this too.

Here is the video with your app without this fixed width:
https://www.loom.com/share/488ec7addd2345dc921aa1db417daacc

And here is again the video where you can see how your app works with the fixed width:
https://www.loom.com/share/74dfeadd52e948e8a5271e502564e8cf